tests/fixtures/descriptor_tests/with_create_descriptor_hook/core/hooks/pick_environment.py

```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2013 Shotgun Software Inc.
+# Copyright (c) 2018 Shotgun Software Inc.
```

Comment:
This is the pick_environment hook used by the config. Github apparently confused this for a rename. Weird.